ethereum/contracts/bridge/L1ERC20Bridge.sol

```diff
@@ -350,10 +453,16 @@ contract L1ERC20Bridge is IL1Bridge, IL1BridgeLegacy, AllowListed, ReentrancyGua
     }
 
     /// @return The L2 token address that would be minted for deposit of the given L1 token
-    function l2TokenAddress(address _l1Token) public view returns (address) {
+    function l2TokenAddress(address _l1Token, uint256 _chainId) public view returns (address) {
         bytes32 constructorInputHash = keccak256(abi.encode(address(l2TokenBeacon), ""));
```

Comment:
I am not sure, that `l2TokenBeacon` will be the same on each chain. What do you think?

Reply:
So if the bytecode does not change, then it should be the same, as it can be calculated like so: 
https://github.com/matter-labs/era-contracts/blob/f06a58360a2b8e7129f64413998767ac169d1efd/ethereum/scripts/initialize-bridges.ts#L129

But if we want to change these things, then I will put it into a mapping. But it would be good to have some standards here.

Reply:
This also ties into the cross chain standards. It is possible to have the same contract on the same address on different chains. This would be very useful for hyperbridging. So even if we have multiple versions, I think we should have a few allowed versions instead of each chain having its own.

Reply:
I changed this. Due to security reason, we will have to store the address of the tokens on each chain. We will still have a standard address, which will be specified by the governor.